### PR TITLE
Add 'Lim Chee Aun'

### PR DESCRIPTION
**Pull request by cheeaun**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Amas is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym).
 - [Wenjie Fan](https://github.com/DIYgod/ama) - Front-end Engineer for Linux Deepin, blogger of anotherhome.net.
 - [Maxime Euzière](https://github.com/xem/ama) - Front-end hacker and JS code-golfer.
 - [Robert Haritonov](https://github.com/operatino/ama) - Front-end Team Lead, [SourceJS](http://sourcejs.com) maintainer and Living Style Guide evangelist.
-
```
Comment (arthurvr) [inline readme.md:28]: Don't remove this line :)

Comment (cheeaun) [inline readme.md:28]: Weird. I think GitHub's diff is not doing a good job here :confused: This line wasn't actually removed at all if you compare the raw markdown files.

**Comment by arthurvr** [comment]
> Add an empty list item?

That wasn't there before eighter. Just make sure you have two lines between sections as there was before and don't touch unrelated lines.

Fixed it for you while merging.

**Comment by cheeaun** [comment]
@arthurvr cool thanks :sweat_smile: GitHub's visual diff sure is confusing.

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Pull request closed without merging**